src/Surfnet/StepupRa/RaBundle/Service/InstitutionListingService.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupRa\RaBundle\Service;
+
+use Surfnet\StepupMiddlewareClientBundle\Identity\Service\InstitutionListingService as ApiInstitutionListingService;
+use Surfnet\StepupMiddlewareClientBundle\Identity\Dto\InstitutionListing;
+
+class InstitutionListingService
+{
+    /**
+     * @var ApiInstitutionListingService
+     */
+    private $institutionListingService;
+
+    /**
+     * @param ApiInstitutionListingService $institutionListingService
+     */
+    public function __construct(
+        ApiInstitutionListingService $institutionListingService
+    ) {
+        $this->institutionListingService = $institutionListingService;
+    }
+
+    public function getAll()
```

Comment:
As this method has a mixed result type, adding a type hint might be a good idea. It might be even better to thrown an exception when the method would return false.

Reply:
Refactored so it will always return an array